catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DuplicateCommands.java

```diff
@@ -53,6 +55,12 @@
 
     private static final String DATE_FORMAT = "MM-dd-yyyy";
 
+    private static final String MUTUALLY_EXCLUSIVE_OPTION_MESSAGE =
+            "This option is mutually exclusive with other --lastXXXX options.";
```

Comment:
I'm not sure anyone but us would even know what "mutually exclusive" means. Should probably dumb this down a bit.